package.json

```diff
@@ -29,7 +30,7 @@
   "private": true,
   "scripts": {
     "build": "nuxt build",
-    "dev": "npm run dev:prisma && nuxt dev",
+    "dev": "npx prisma migrate dev && nuxt dev",
```

Comment:
Why this change?

Reply:
Switching from `npm` to `npx` ensures the correct local version of Prisma is used without global installations, promoting consistency and simplifying setup. It also provides immediate feedback on command issues, improving the development workflow.

Reply:
@naaa760, correct me if I am wrong, but I think it makes no difference in this case 💭